sycl/doc/SYCLDriverforRFC.md

```diff
@@ -0,0 +1,53 @@
+# Adjust the compiler driver to support offloading for SYCL kernels
+
+Supporting the SYCL programming model requires a number of adjustments to the
+compiler driver. These changes involve introducing enabling command line
+options, adjusting the compilation tool chains to provide device specific
+compilations as well as corresponding host compilations. The behaviors are
+centered around compilation of source files, consumption of objects and
+libraries that contain SYCL kernels will be done by the `clang-linker-wrapper`.
+
+## Command line options for enabling.
+
+A number of command line options will be introduced to enable the offloading
+compilation for SYCL.
+* -fsycl
+  * Enables the device and host compilations against a given source file.
+* -fsycl-targets=\<target\>
+  * Compiles the device based on the given target value. 
+* -fsycl-device-only
+  * Enables the ability to create only device code.
+* -fsycl-host-compiler
+  * Provide the ability to use a 3rd party compiler to perform the host
+    compilation step.
+
+## Introduction of SYCL specific offload action builder
+
+An additional offloading target toolchain will be created to support the SYCL
+device compilations. This will be unique to the existing OpenMP and other
+offloading targets allowing for the creating of unique toolchains specific
+for each SYCL based target being compiled for.
+
+## Implementation of two compilation modes
```

Comment:
This section seems confusing to me because it makes it sound like `-fsycl-device-only` is an alternative to the multi-pass compiler approach.  But this is not the case because you cannot even use `-fsycl-device-only` to produce a runnable application.

What is the message you want to convey to the audience with this section?

Reply:
I've changed the section to provide more details of the various compilation modes enabled via the new options provided.  Hopefully it is clearer.

Reply:
Thanks, much better!